CONTRIBUTING.md

```diff
@@ -346,3 +375,31 @@ Even minor versions are dedicated to official releases, e.g. `*.6.*`.
 ### Playground
 
 - [run the playground locally](/website/playground/README.md)
+
+## Releasing
+
+When releasing a new version of a Biome, follow these steps:
+
+1. [ ] Add a [changelog](./CHANGELOG.md) entry for every Pull Request that lacks one.
+   You can filter [merged PRs that don't update the changelog](https://github.com/biomejs/biome/pulls?q=is%3Apr+is%3Amerged+-label%3AA-Changelog).
+   Read our [guidelines for editing the changelog](#changelog).
+
+1. [ ] Based on the [changelog](./CHANGELOG.md), determine which version number to use.
+   See our [versioning guide](https://biomejs.dev/internals/versioning/) for more details.
+
+1. [ ] Rename `Unreleased` to `<version> (iso-date)` in the [changelog](./CHANGELOG.md).
+   Then update the website using `cargo codegen-website`.
```

Comment:
```suggestion
1. [ ] Rename `Unreleased` to `<version> (iso-date)` in the [changelog](./CHANGELOG.md).
   Then update the website using `BIOME_VERSION=<version> cargo codegen-website`.
```

This is important, with `BIOME_VERSION`, the task will generate the new JSON schema